skimage/filters/_fft_based.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import functools
+from .._shared.fft import fftmodule as fft
+from ..exposure import rescale_intensity
+from ..util import dtype_limits
+
+
+def _get_ND_butterworth_filter(shape, factor, order, high_pass, real):
+    """Create a N-dimensional Butterworth mask for an FFT
+
+    Parameters
+    ----------
+    shape : tuple of int
+        Shape of the n-dimensional FFT and mask.
+    factor : float
+        Fraction of mask dimensions where the cutoff should be.
+    order : float
+        Controls the slope in the cutoff region.
+    high_pass : bool
+        Whether the filter is high pass (low frequencies attenuated) or
+        low pass (high frequencies are attenuated).
+    real : bool
+        Whether the FFT is of a real (True) or complex (False) image
+
+    Returns
+    -------
+    wfilt : ndarray
+        The FFT mask.
+    """
+    ranges = []
+    for i, d in enumerate(shape):
+        # start and stop ensures center of mask aligns with center of FFT
+        axis = np.arange(-(d - 1) // 2, (d - 1) // 2 + 1) / (d * factor)
+        ranges.append(fft.ifftshift(axis ** 2))
+    # for real image FFT, halve the last axis
+    if real:
+        limit = d // 2 + 1
+        ranges[-1] = ranges[-1][:limit]
+    # q2 = squared Euclidian distance grid
+    q2 = functools.reduce(
+            np.add, np.meshgrid(*ranges, indexing="ij", sparse=True)
+            )
+    # division of order by 2 to avoid additional square root on q2
+    wfilt = 1 / np.sqrt(1 + np.power(q2, order / 2))
```

Comment:
```suggestion
    wfilt = 1 / np.sqrt(1 + np.power(q2, order))
```


Assuming this `order` variable corresponds to `n` in the first equation of the [Butterworth wikipedia article](https://en.wikipedia.org/wiki/Butterworth_filter), then it seems this should be `order` rather than `order / 2`? I realize that you have avoided taking a sqrt, but it seems this would only reduce the term from (`2 * order`) to `order` and not to `order / 2`